packages/owl-bot/src/owl-bot.ts

```diff
@@ -49,11 +50,23 @@ interface PubSubContext {
   };
 }
 
-export function OwlBot(
-  privateKey: string | undefined,
-  app: Probot,
-  db?: Db
-): void {
+async function acquireLock(target: string): Promise<DatastoreLock | boolean> {
+  const lock = new DatastoreLock('owlbot', target, 25 * 1000);
```

Comment:
Perhaps we should put the time into a constant and add also add it as an optional parameter to the function

Reply:
I think starting with a `const` is a good idea, makes it more readable. I don't think there's a good reason to configure it differently for different calls at this time, so I wouldn't bother making it an argument.